lib/std/std.zig

```diff
@@ -39,6 +39,10 @@ pub const StringHashMap = hash_map.StringHashMap;
 pub const StringHashMapUnmanaged = hash_map.StringHashMapUnmanaged;
 pub const StringArrayHashMap = array_hash_map.StringArrayHashMap;
 pub const StringArrayHashMapUnmanaged = array_hash_map.StringArrayHashMapUnmanaged;
+pub const SymbolMapDarwin = @import("symbol_map_darwin.zig").SymbolMap;
+pub const SymbolMapUnix = @import("symbol_map_unix.zig").SymbolMap;
+pub const SymbolMapPDB = @import("symbol_map_pdb.zig").SymbolMap;
+pub const SymbolMapUnsupported = @import("SymbolMapUnsupported.zig");
```

Comment:
Right now these are part of the public interface. I think this is reasonable because these types (particularly SymbolMapUnix) would be useful for [byos](https://github.com/ziglang/zig/issues/3784).